#45381: .toWeb()/.fromWeb() added to webstreams.md file

jayshreek3 opened the pull request:
…streams docs fixed

<!--
Before submitting a pull request, please read:

- the CONTRIBUTING guide at https://github.com/nodejs/node/blob/HEAD/CONTRIBUTING.md
- the commit message formatting guidelines at
  https://github.com/nodejs/node/blob/HEAD/doc/contributing/pull-requests.md#commit-message-guidelines

For code changes:
1. Include tests for any bug fixes or new features.
2. Update documentation if relevant.
3. Ensure that `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes.

If you believe this PR should be highlighted in the Node.js CHANGELOG
please add the `notable-change` label.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
-->

aduh95 commented:
Hey, thanks for the PR! Can you please fix the linter errors, and use a commit message that match our guidelines: https://github.com/nodejs/node/blob/1149d6b49de990f70285fee61c02881e471069bc/doc/contributing/pull-requests.md#L164-L191

jayshreek3 commented:
Hey @aduh95, thank you for reviewing my earlier PR. I have fixed the linter issues and added appropriate comment for the commit message. Please take a look

aduh95 reviewed (commented):
Can you please remove all the unrelated changes? If you look at the code diff, it should contain only changes necessary to fix the linked issue, nothing more.

jayshreek3 commented:
> Can you please remove all the unrelated changes? If you look at the code diff, it should contain only changes necessary to fix the linked issue, nothing more.

Have now kept only the relevant code changes. @aduh95 Kindly checkout the new PR #50328, and you may close PR #50255
